cypress/e2e/notes-100/access-shared-note.cy.js

```diff
@@ -1,16 +1,39 @@
 import {waitForModel, homepageSetup} from '../../support/utils'
 
-
-describe('select-a-note', () => {
-  context('Open the link with a shared note', () => {
+// From https://github.com/bldrs-ai/Share/issues/1072
+describe('Notes 100 - Access shared note', () => {
+  context('Access shared link containing the path to the general notes', () => {
     beforeEach(() => {
       homepageSetup()
-    })
-    it('Visit notes permalink, side drawer shall be opened', () => {
       cy.setCookie('isFirstTime', '1')
       cy.visit('/share/v/p/index.ifc#c:-133.022,131.828,161.85,-38.078,22.64,-2.314;i:')
       waitForModel()
+    })
+    it('Panel title to contain NOTES string', () => {
```

Comment:
Still needs percySnapshot.

On mine I've been noticing they're running slow.

What do you think about this instead?

```
it('Correct project to be loaded into the viewport and side drawer to be open - Screen', () => {
  // Panel title to contain 'NOTES' string
  cy.get('[data-testid="panelTitle"]').contains('NOTES')

  // List of notes to be visible
  cy.get('.MuiList-root').should('exist')
  
  cy.percySnapshot()
})

Reply:
That's cool, I was thinking we should have 'IT' for each expect, but it makes sense to combine these two ITs together since it points to the setup and the screenshot should capture the baseline state.

Reply:
I like this format, I suppose it is judgement call, also we should include comments in this format?

Reply:
Comments help when the selectors are generic, like .MuiList-root is ambiguous to me without going and reading the code under test and Mui api to remember how this works.  e2e tests should be more black-box, which is why i've been moving towards data-testids that reflect use-perceivable types